sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultSyncPoller.java

```diff
@@ -37,42 +37,38 @@
     /**
      * Creates DefaultSyncPoller.
      *
-     * @param defaultPollInterval the default polling interval
-     * @param activationOperation the activation operation to be invoked at most once across all subscriptions,
-     *                            this parameter is required, if there is no specific activation work to be
-     *                            done then invocation should return Mono.empty(), this operation will be called
-     *                            with a new {@link PollingContext}.
+     * @param pollInterval the polling interval
+     * @param syncActivationOperation the operation to synchronously activate (start) the long running operation,
+     *     this operation will be called with a new {@link PollingContext}.
      * @param pollOperation the operation to poll the current state of long running operation, this parameter
-     *                      is required and the operation will be called with current {@link PollingContext}.
-     * @param cancelOperation a {@link Function} that represents the operation to cancel the long-running operation
-     *                        if service supports cancellation, this parameter is required and if service does not
-     *                        support cancellation then the implementer should return Mono.error with an error message
-     *                        indicating absence of cancellation support, the operation will be called with current
-     *                        {@link PollingContext}.
+     *     is required and the operation will be called with current {@link PollingContext}.
+     * @param cancelOperation a {@link Function} that represents the operation to cancel the long running operation
+     *     if service supports cancellation, this parameter is required and if service does not support cancellation
```

Comment:
Break some of these sentences up - in a few places you flow straight into "this parameter is required" when a full stop and a new sentence would improve readability.